app/src/main/java/com/sunlightlabs/congress/models/Bill.java

```diff
@@ -16,71 +16,65 @@
 	// basic
 	public String id, bill_type, chamber;
 	public int congress, number;
-	
+
+    // TODO: chop down to just title
 	public String short_title, official_title;
-	public Date last_action_at, last_passage_vote_at;
-	public int cosponsors_count;
-	
-	public Date introduced_on, house_passage_result_at, senate_passage_result_at;
-	public Date vetoed_at, house_override_result_at, senate_override_result_at;
-	public Date senate_cloture_result_at;
-	public Date awaiting_signature_since, enacted_at;
-	
-	public boolean vetoed, awaiting_signature, enacted;
-	public String house_passage_result, senate_passage_result, house_override_result, senate_override_result;
-	public String senate_cloture_result;
-	
-	// new history field: whether and when a bill became unusually active
-	public boolean active;
-	public Date active_at;
-	
+    public Date last_action_on;
+
+    public Date introduced_on, house_passage_result_on, senate_passage_result_on;
+	public Date vetoed_on, enacted_on;
+    public boolean vetoed, enacted;
+
+    // whether a bill made it past routine introduction/referral
+    public boolean active;
+    public int cosponsors_count;
+
+    // summary
+    public String summary;
+
+    // official URLs
+    public String govtrack_url, congress_url, gpo_url;
+
 	// sponsor
 	public Legislator sponsor;
-	
+
 	// cosponsors
 	public List<Legislator> cosponsors;
-	
-	// summary
-	public String summary;
-	
+
 	// votes
 	public List<Bill.Vote> votes;
 	
 	// actions
 	public List<Bill.Action> actions;
 	public Bill.Action lastAction;
-	
-	// search result metadata (if coming from a search)
-	public SearchResult search;
-	
-	// homepage URLs on services
-	public Map<String,String> urls;
-	
-	// full text URLs (to GPO)
-	// valid keys: "html", "xml", "pdf"
-	public Map<String,String> versionUrls;
+
 
 	public static class Action implements Serializable {
 		private static final long serialVersionUID = 1L;
 
-		// required
-		public String type, text;
-		public Date acted_at;
-		
-		// optional
-		public String chamber, result;
+		public String type, description, chamber;
+		public Date acted_on;
 	}
 	
 	public static class Vote implements Serializable {
 		private static final long serialVersionUID = 1L;
-		//todo: rename passage_type->vote_type
-		public String result, text, how, passage_type, chamber, roll_id;
-		public Date voted_at;
+		public String roll_id, result, question, chamber;
+        public int yes, no, not_voting;
```

Comment:
We're not showing these new fields of `yes`, `no`, and `not_voting` anywhere, but capturing them here anyway.